config.go

```diff
@@ -116,6 +117,7 @@ type config struct {
 	GetWorkKeys        []string      `long:"getworkkey" description:"DEPRECATED -- Use the --miningaddr option instead"`
 	AddrIndex          bool          `long:"addrindex" description:"Build and maintain a full address index. Currently only supported by leveldb."`
 	DropAddrIndex      bool          `long:"dropaddrindex" description:"Deletes the address-based transaction index from the database on start up, and the exits."`
+	SigCacheMaxSize    int           `long:"sigcachemaxsize" description:"hello"`
```

Comment:
Fixed. 

The `hello` was left over from when I just needed it all to compile :)